docs/developer_guide/testing_suite.rst

```diff
@@ -29,6 +29,10 @@ Then simply run all tests with pytest
 
   pytest
 
+.. note::
+
+  You will likely observe many failed tests at this point, if you have not configured the other aspects of the testing suite shown below.
```

Comment:
```suggestion
  You will likely observe many failed tests if the test data is not available. See the section 'Testing on Example Data' for instructions on how to download the test data.
```

Reply:
I kept it more general because you'll also see failures if you haven't done `neuroconv[full]`, right?

Reply:
Oh, but in the first section we do specify that. OK